src/main/java/core/basesyntax/Number.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Number {
+    public int randomNumberForBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        return randomNumber;
+    }
+}

```

Comment:
no need to create extra classes
```suggestion
```

Reply:
how is it, during the tests it was indicated to me that the number and color should be private, and if you remove the getters, how to get the information?